packages/react-component-library/src/components/Table/Table.tsx

```diff
@@ -26,6 +26,11 @@ export interface TableProps extends ComponentWithClass {
    * Optional text caption to display above the component.
    */
   caption?: string
+  /**
+   * Optional, callback when a row is clicked
```

Comment:
```suggestion
   * Optional handler invoked when a row is clicked.
```

Reply:
Sorry I know this is really nitpicky - just trying to align it with style of other prop interface comments (tone of voice / full stop).